pkg/proctree/proctree_procfs.go

```diff
@@ -158,8 +158,9 @@ func dealWithProc(pt *ProcessTree, givenPid int32) error {
 
```

Comment:
should `getProcessByPID` also use epochTimeNs? https://github.com/aquasecurity/tracee/blob/38833cc906466d311365afcdb02ec03cc4962549/pkg/proctree/proctree_procfs.go#L110-L112